juego.py

```diff
@@ -66,87 +70,101 @@ def __init__(self):
         pass
 
 
+    global sx, sy
+    def sx(coord_x):
+        return coord_x * scale_x
+
+    def sy(coord_y):
+        return coord_y * scale_y
+
     def main(self):
-        sonido_menu = load_sound("./data/menu.ogg")
+        sonido_menu = load_sound("menu.ogg")
```

Comment:
load_sound should (but does not) use get_activity_root; test the activity when installed in /usr/share/sugar/activities

Reply:
Done in 01ea1b9. Also tested; no errors